packages/preload/src/modules/workspace.ts

```diff
@@ -333,3 +335,11 @@ export async function openFolder(_path: string) {
     throw new Error(error);
   }
 }
+
+export async function npmPackageOutdated(_path: string, _package: string): Promise<boolean> {
+  return invoke('bin.npmPackageOutdated', _path, _package);
+}
+
+export async function installNpmPackage(_path: string, _package: string): Promise<void> {
+  return invoke('bin.installNpmPackages', _path, _package);
+}
```

Comment:
Same as above, the `workspace` module revolves around the `Project` type (creating/duplicating/listing/deleting projects), I would move these two function to a separate `npm` module.

We might want to consider renaming eventually `workspace` into `projects` or something less vague in the future, but not for this PR.